v2/stacks/dataset-catalogue/Makefile

```diff
@@ -1,8 +1,14 @@
 # Include common make targets
 include ../common.mk
 
+# Specifies the default folder where the keys are stored in
+DEFAULT_KEYS_FOLDER ?= ../../../../dis-authentication-stub/static/keys
+
 .PHONY: init
 init: base-init
 	# Add stack specific initialisation logic here
+	gpg -d $(DEFAULT_KEYS_FOLDER)/public.key.asc > $(DEFAULT_KEYS_FOLDER)/public.key
+	gpg -d $(DEFAULT_KEYS_FOLDER)/private.key.asc > $(DEFAULT_KEYS_FOLDER)/private.key
+	gpg -d $(DEFAULT_KEYS_FOLDER)/jwt-keys.json.asc > $(DEFAULT_KEYS_FOLDER)/jwt-keys.json
```

Comment:
I get an error when I run the `make up` command that says:

> # Add stack specific initialisation logic here
> gpg -d ../../../../dis-authentication-stub/static/keys/public.key.asc > ../../../../dis-authentication-stub/static/keys/public.key
> gpg: encrypted with 4096-bit RSA key, ID D54A57C3934B1E3E, created 2021-11-01
>       "ons-sandbox"
> gpg -d ../../../../dis-authentication-stub/static/keys/private.key.asc > ../../../../dis-authentication-stub/static/keys/private.key
> gpg: encrypted with 4096-bit RSA key, ID D54A57C3934B1E3E, created 2021-11-01
>       "ons-sandbox"
> gpg -d ../../../../dis-authentication-stub/static/keys/jwt-keys.json.asc > ../../../../dis-authentication-stub/static/keys/jwt-keys.json
> gpg: can't open '../../../../dis-authentication-stub/static/keys/jwt-keys.json.asc': No such file or directory
> gpg: decrypt_message failed: No such file or directory
> make: *** [init] Error 2
>

Reply:
I got this working but I had to use the fully qualified path to my folder rather than the ../../ I think there should be an update to the README.md in this folder to explain that this environment variable might need to be set to the fully qualified path if this doesn't work.

Reply:
Added some explanations to the readme. :-)